package.json

```diff
@@ -5,7 +5,7 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn workspaces foreach run build",
+    "build": "yarn build-libs && yarn build-samples",
```

Comment:
Success of building sample packages depends on the success of building lib packages.

This change allows the top-level build script to fail early, instead of attempting to build all the packages.